dev/io.openliberty.security.oidcclientcore.internal/resources/io/openliberty/security/oidcclientcore/internal/resources/OidcClientCoreMessages.nlsprops

```diff
@@ -48,3 +48,7 @@ ERROR_BUILDING_AUTHORIZATION_ENDPOINT_URL.useraction=See the error in the messag
 AUTHENTICATION_RESPONSE_ERROR=CWWKS2407E: The {0} OpenID Connect client encountered the following error while verifying the authentication response from the OpenID Connect provider: {1}
 AUTHENTICATION_RESPONSE_ERROR.explanation=The response from the OpenID Connect provider might be missing information. There might be a problem with the code returned from the OpenID Connect provider.
 AUTHENTICATION_RESPONSE_ERROR.useraction=See the error in the message for more information.
+
+TOKEN_VALIDATION_EXCEPTION=CWWKS2408E: The {0} OpenID Connect client encountered the following error while validating the token received from the OpenID Connect provider: {1}
```

Comment:
Hi Aruna,
Adam is using the message id CWWKS2408E in https://github.com/OpenLiberty/open-liberty/pull/22399/files#diff-1716839aaee06547dff34a3b50bf9b171294dc76a10f4942fbcb596dc9c8dd90 for his PR. Please review his changes or coordinate with Adam to choose another message id. Thanks!

Reply:
I will update the message id and message in another PR